debian/rules

```diff
@@ -13,10 +12,9 @@ override_dh_python3:
 
 override_dh_missing:
 	rm -r `find debian/tmp -name __pycache__`
-	dh_missing --fail-missing
 
 get-orig-source:
```

Comment:
native package does not have any original tarball. SO not sure what this target is for at all.

do we publish upload sdist tarball to like pypi?

Reply:
I don't know, I'm +1 for dropping it. It's even deprecated since Debian policy 4.1.4. If it's a helper target for publishing somewhere else then it doesn't belong to `debian/` I'd say.

Reply:
Let's first get master up to date, then I'll drop it with some other changes I have done locally.